src/components/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import Accordion from "."
+import { Meta, StoryObj } from "@storybook/react"
+import { VStack } from "@chakra-ui/react"
+
+type AccordionType = typeof Accordion
+
+const meta: Meta<AccordionType> = {
+  title: "Molecules / Disclosure Content / Accordions",
+  component: Accordion,
+  args: {
+    label: "Accordion label",
```

Comment:
This is another item to match the figma file, where we should be using the same text.

```suggestion
    label: "Label text of the accordion",
```